feature/mpls/sr/otg_tests/srte_egress/egress_node_forwarding /README.md

```diff
@@ -0,0 +1,190 @@
+# SR-1.2: Egress Node Forwarding
+
+## Summary
+
+This is to test the forwarding functionality of MPLS traffic on an Egress node 
+in SRTE+MPLS enabled network.
+
+The tests validate that the DUT performs the following actions -
+
+ - DUT is an egress node in SRTE+MPLS network.
+ - DUT will receive unlabled traffic for IPv4 destination and perform IPv4 forwarding.
+ - DUT wil receive MPLS Label 2 traffic for IPv6 destinations.
+ - DUT will pop the MPLS label 2 and perform IPv6 forwarding.
+
+
+## Testbed type
+
+* [`featureprofiles/topologies/atedut_2.testbed`](https://github.com/openconfig/featureprofiles/blob/main/topologies/atedut_2.testbed)
+
+## Procedure
+
+### Test environment setup
+
+* Create the following connections:
+* DUT has ingress and egress port connected to the ATE.
+  
+```mermaid
+graph LR; 
+A[ATE:Port1] --Ingress--> B[Port1:DUT:Port2];B --Egress--> C[Port2:ATE];
+```
+
+* ATE Port 1 hosted prefixes:
+  
+  * ATE-Port1 IPV4 address = ATE-P1-Address
+  * Additional Source Address = IPV4-SRC1
+  * Additional Source Address = IPV6-SRC1
+
+* ATE Port 2 hosted prefixes:
+  
+  * ATE-Port2 IPV4 address = ATE-P2-Address
+  * Additional destination address = IPV4-DST1
+  * Additional destination address = IPV6-DST1
+
+*  ATE Port 1 generates below flow types:
+ 
+ * Flow type 1:  Ethernet+IPv4+Payload
+  * For the Ethernet Header:
+     * Source MAC address: Unicast Ethernet MAC
+     * Destination MAC address: DUT-Port1 mac address
+   * For the IP header:
+     * Source IP and Destination IP will be IPV4-SRC1 and IPV4-DST1 respectively.
+     * Protocol can be TCP/UDP and source and destination port will vary.
+
+ * Flow type 2:  Ethernet+MPLS+IPv6+Payload
+  * For the Ethernet Header:
+     * Source MAC address: Unicast Ethernet MAC
+     * Destination MAC address: DUT-Port1 mac address
+  * For MPLS header:
+     * MPLS label 2
+     * EXP set to 0
+     * S set to 1
+     * TTL set to 64  
+  * For the IPv6 header:
+     * Source IP and Destination IP will be IPV6-SRC1 and IPV6-DST1 respectively.
+     * Protocol can be TCP/UDP and source and destination port will vary.
+## Procedure
+
+### Configuration
+                              
+*   Configure Segment Routing Global Block (srgb) lower-bound: 400000 upper-bound: 465001)
+*   Enable Segment Routing for the ISIS
```

Comment:
Where is IS-IS running in this test? Do you expect that the ATE ports are neighbours of the DUT? If so, please add something stating this.

Reply:
Re-opening again -- should this just be removed? What are we trying to enable here?